.yarn/releases/yarn-3.2.3.cjs

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 /* eslint-disable */
 //prettier-ignore
```

Comment:
This file should probably be ignored
Maybe try to ignore it in `.prettierignore` or replace the prettier-ignore comment with
`/* prettier-ignore */`